packages/core/package.json

```diff
@@ -59,10 +59,6 @@
     "fix:prettier": "prettier . --write",
     "fix:lint": "eslint src *.js --fix",
     "pretest": "run-s pretest:*",
-    "pretest:1-init-git-submodules": "[ -f '../../nwaku/build/wakunode2' ] || git submodule update --init --recursive",
-    "pretest:2-build-nwaku": "[ -f '../../nwaku/build/wakunode2' ] || run-s nwaku:build",
-    "nwaku:build": "(PROC=$(nproc --all 2>/dev/null || echo 2); cd ../../nwaku; make -j$PROC update; NIMFLAGS=\"-d:chronicles_colors=off -d:chronicles_sinks=textlines -d:chronicles_log_level=TRACE\" make -j$PROC wakunode2)",
```

Comment:
The docker image are not build with these options so logs contain some color or other random chars:

```
      �TRC 2023-03-29 04:42:56.808+00:00 registering protocols                      topics="libp2p multistream" tid=1 file=multistream.nim:224 protos="@[\"/libp2p/autonat/1.0.0\"]"
```

@jm-clius do we have an option to pass to `wakunode2` to get clean ASCII logs?

Reply:
Unfortunately compiled binary can only set `logLevel` and `logFormat` (as TEXT or JSON).